README.md

```diff
@@ -32,15 +32,25 @@ Install all dependencies:
 yarn
 ```
 
-and start dev-server:
+and start dashboard locally:
 
-The start command requires to specify a remote Eclipse Che server like:
+```sh
+yarn start
```

Comment:
is `yarn` really required before `yarn start`?

Reply:
Yes, it is.

Reply:
We need to install all dependencies first.